src/data_race.rs

```diff
@@ -687,15 +715,12 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: MiriEvalContextExt<'mir, 'tcx> {
         }
     }
 
-    fn reset_vector_clocks(
```

Comment:
all changes below this line in this file are rustfmt changes. Did rustfmt not pick these up in the first commit?

Reply:
Hmm, that must have been the case